coordinator/src/services/communication/tracker/tracker-response-builder.ts

```diff
@@ -53,7 +53,7 @@ const createTrackerResponse = (
 ): TrackerResponse => {
   const prescription = extractPrescription(spineResponse.body, logger)
   if (prescription) {
-    logger.info(`Successfully extracted prescription ${prescription.id}`)
+    logger.info(`Successfully extracted prescription ${prescription.id._attributes.root}`)
```

Comment:
Trivial unrelated change, just a logging error found while doing this ticket's work